tests/testthat/test-input-checks.R

```diff
@@ -0,0 +1,65 @@
+
+setup_test_results <- function() {
+  test_results <- tibble::tribble(
+    ~TestId, ~TestName, ~passed, ~failed,
+    "FOO-BAR-001", "t1", 2, 0,
+    "FOO-BAR-002", "t2", 1, 0,
+    "FOO-BAR-003", "t3", 1, 0,
+    NA, "t4", 3, 0)
+  readr::write_csv(test_results, "t.csv")
```

Comment:
this should probably clean up after itself. Maybe use `withr::defer()` and pass the parent environment (so when the function/environment that calls this exits, it will delete the files it wrote). 

**Update:** oh I see, you're just using it in a tempdir. I guess this is fine then. Something to think about though.

Reply:
I'll add a comment documenting that this expects be called from within `withr::with_tempdir` or the like.